.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npx lint-staged
+ggshield secret scan pre-commit
```

Comment:
nit: it's better to add a newline at the end of the file

Reply:
done